lib/fastlyctl/commands/tls/managed.rb

```diff
@@ -0,0 +1,119 @@
+module FastlyCTL
+  class TLSManagedSubCmd < SubCommandBase
+    SubcommandPrefix = "tls managed"
+    DomainRegex = /(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]/
+
+    desc "create <domain>", "Create a Fastly Managed TLS Subscription for [domain]. A Certificate will be requested from lets-encrypt once you satisfy one of the challenges. You can learn more about the challenge types here: https://letsencrypt.org/docs/challenge-types/ and Fastly's API documentation here: https://docs.fastly.com/api/tls-subscriptions."
+    def create(domain)
+      abort "Must specify valid domain name" unless domain =~ DomainRegex
+
+      tls_configs = FastlyCTL::TLSUtils.get_tls_configs
+      tls_config = FastlyCTL::TLSUtils.select_tls_config(tls_configs)
```

Comment:
Allow this to be done by cli parameter?

Reply:
Ah, good call. I will add that during the next pass.